examples/webgl2_multiple_rendertargets_multisampled.html

```diff
@@ -0,0 +1,339 @@
+
+				vUv = uv;
+				gl_Position = projectionMatrix * modelViewMatrix * vec4( position, 1.0 );
+
+			}
+		</script>
+		<script id="render-frag" type="x-shader/x-fragment">
+			precision highp float;
+			precision highp int;
+
+			layout(location = 0) out vec4 pc_FragColor;
+
+			in vec2 vUv;
+
+			uniform sampler2D tDiffuse;
+			uniform sampler2D tNormal;
+
+			void main() {
+
+				vec3 diffuse = texture( tDiffuse, vUv ).rgb;
+				vec3 normal = texture( tNormal, vUv ).rgb;
+
+				pc_FragColor = vec4(vec3(0.), 1.);
+				pc_FragColor.rgb += mix( diffuse, normal, step( 0.5, vUv.x ) );
+
+			}
+		</script>
+
+	</head>
+	<body>
+		<div id="info">
+			<a href="http://threejs.org" target="_blank">threejs</a> - WebGL - Multiple Render Targets Multisampled<br/>
+			Created by <a href="http://twitter.com/onirenaud" target="_blank">@onirenaud</a> with the help of <a href="http://twitter.com/Cody_J_Bennett" target="_blank">@Cody_J_Bennett</a>.
+		</div>
+		<div id="container">
+		</div>
+		<!-- Import maps polyfill -->
+		<!-- Remove this when import maps will be widely supported -->
+		<script async src="https://unpkg.com/es-module-shims@1.3.6/dist/es-module-shims.js"></script>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js"
+				}
+			}
+		</script>
+
+		<script type="module">
+
+			import * as THREE from 'three';
+
+			import WebGL from './jsm/capabilities/WebGL.js';
+			import { OrbitControls } from './jsm/controls/OrbitControls.js';
+			import { GUI } from './jsm/libs/lil-gui.module.min.js';
+			import Stats from './jsm/libs/stats.module.js';
+
+			let camera, scene, renderer, controls, container, group;
+			let renderTarget;
+			let postScene, postCamera;
+			let stats;
+
+			const clock = new THREE.Clock();
+			// Create a multi render target with Float buffers
+			const gui = new GUI();
+
+			const effectController = {
+				msaa: true,
+			};
+
+			init();
+			gui.add( effectController, 'msaa', true ).onChange( () => {
+
+				renderTarget.samples = effectController.msaa ? 4 : 0;
+				render();
+
+			} );
+
+			function init() {
+
+				container = document.getElementById( 'container' );
+
+				if ( WebGL.isWebGL2Available() === false ) {
+
+					document.body.appendChild( WebGL.getWebGL2ErrorMessage() );
+					return;
+
+				}
+
+				renderer = new THREE.WebGLRenderer();
+				renderer.setPixelRatio( 1 );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				container.appendChild( renderer.domElement );
+
+				stats = new Stats();
+				stats.domElement.style.position = 'absolute';
+				stats.domElement.style.top = '0px';
+				container.appendChild( stats.domElement );
+
+
+				renderTarget = new THREE.WebGLMultipleRenderTargets(
+					window.innerWidth,
+					window.innerHeight,
+					2,
+					{
+						samples: effectController.msaa ? 4 : 0
+					}
+				);
+				renderTarget.depthTexture = new THREE.DepthTexture( 512, 512 );
+				renderTarget.depthTexture.type = THREE.FloatType;
```

Comment:
I don't see this used in the example anywhere. Just testing depth/stencil?

Reply:
When using MRT in threejs that is the only way to get a proper depth buffer, otherwise you will get weird behaviors when two geometries overlap 🤷